community/kernel-api/src/test/java/org/neo4j/internal/kernel/api/NodeLabelIndexCursorTestBase.java

```diff
@@ -36,17 +37,25 @@
     @Override
     void createTestGraph( GraphDatabaseService graphDb )
     {
+        Node toDelete;
         try ( Transaction tx = graphDb.beginTx() )
         {
             graphDb.createNode( label( "One" ), label( "First" ) );
             graphDb.createNode( label( "Two" ), label( "First" ) );
             graphDb.createNode( label( "Three" ), label( "First" ) );
+            toDelete = graphDb.createNode( label( "One" ) );
             graphDb.createNode( label( "Two" ) );
             graphDb.createNode( label( "Three" ) );
             graphDb.createNode( label( "Three" ) );
 
             tx.success();
         }
+
+        try ( Transaction tx = graphDb.beginTx() )
+        {
+            toDelete.delete();
+            tx.success();
+        }
     }
```

Comment:
What is the reason for that change? To have an id that is known for being empty later?

Reply:
To make sure that the label-scan doesn't return deleted nodes, it was in one of the `KernelIT` tests.